cmd/Cargo.toml

```diff
@@ -7,15 +7,15 @@ publish = false
 default-run = "tikv-server"
 
 [features]
-default = ["tikv/default"]
 tcmalloc = ["tikv/tcmalloc"]
 jemalloc = ["tikv/jemalloc"]
 mimalloc = ["tikv/mimalloc"]
 portable = ["tikv/portable"]
 sse = ["tikv/sse"]
 mem-profiling = ["tikv/mem-profiling"]
-profiling = ["tikv/profiling"]
```

Comment:
Why remove this feature?

Reply:
It is moved to tests/Cargo.toml because profiling is only used by the profiler crate which is only used via the tests entry point.

Reply:
Interesting. I thought it was going to support profiling tikv-server at runtime. But I'm OK to readd this when it's implemented.